src/notificationcenter.cs

```diff
@@ -103,14 +103,21 @@ interface NSExtensionContext_NCWidgetAdditions {
 	[BaseType (typeof (UIVibrancyEffect))]
 	interface UIVibrancyEffect_NCWidgetAdditions {
 		[iOS (10,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'UIVibrancyEffect.GetWidgetEffectForVibrancyStyle' instead. ")]
 		[Static]
 		[Export ("widgetPrimaryVibrancyEffect")]
 		UIVibrancyEffect GetWidgetPrimaryVibrancyEffect ();
 
 		[iOS (10,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'UIVibrancyEffect.GetWidgetEffectForVibrancyStyle' instead. ")]
 		[Static]
 		[Export ("widgetSecondaryVibrancyEffect")]
 		UIVibrancyEffect GetWidgetSecondaryVibrancyEffect ();
+
+		[iOS (13,0)]
+		[Static]
+		[Export ("widgetEffectForVibrancyStyle:")]
+		UIVibrancyEffect GetWidgetEffectForVibrancyStyle (UIVibrancyEffectStyle vibrancyStyle);
```

Comment:
Usually we shorten the C# name when the first parameter name contains some of the information from the selector, since it makes it nicer in C#:

```suggestion
		UIVibrancyEffect GetWidgetEffect (UIVibrancyEffectStyle vibrancyStyle);
```